src/components/views/settings/tabs/user/SessionManagerTab.tsx

```diff
@@ -61,6 +80,11 @@ const useSignOut = (
         if (!deviceIds.length) {
             return;
         }
+        const userConfirmedSignout = await confirmSignOut(deviceIds.length);
```

Comment:
The promise is not catch in case of reject, is-it relevant to catch it here ?

Reply:
Modal shouldn't throw any errors outside of exceptional circumstances so I think defaulting to the nearest error boundary is the right thing to do here.